packages/components/src/vue/components/morph-parts/inflections-list.vue

```diff
@@ -4,14 +4,15 @@
 
             <span class="alpheios-inflections-list__inflset_index" v-if="inflections.length > 1">{{ ifindex + 1 }}.</span>
 
-            <div class="alpheios-inflections-list__forms">
+      <div class="alpheios-inflections-list__forms">
                 <span
                     v-for="feat in featuresList.wordParts.filter(feat => inflset.groupingKey[feat.name])" v-bind:key="feat.name"
                     :lang="languageCode" class="alpheios-inflections-list__formtext"
                     data-grouplevel="1" data-feature="feat.name"
                 >{{ feat.template.replace('%s', inflset.groupingKey[feat.name]) }}</span>
+        <annotation-word-form></annotation-word-form>
```

Comment:
So we can't use the data-annotation-type attribute here because the annotation component is drawn first?

Reply:
Yes, since the `AnnotationArea` is the outer component, it is mounted first. Once it's loaded, Vue loads inner components, such as `AnnotationWordForm`. Because of this, to instantiate an `AnnotationWordForm` from the  `AnnotationArea.mount()` would be too early. But maybe there is some other way to handle this.